FHEM/98_LEDController.pm

```diff
@@ -982,7 +982,13 @@ sub LEDController_BuildWebCmd($) {
             # If options are available, use them
             if(defined($field->{options}) && ref($field->{options}) eq 'ARRAY') {
                 for my $i (0..$#{$field->{options}}) {
-                    push @options, "$i," . $field->{options}->[$i];
+                    my $option = $field->{options}->[$i];
+                    # Quote option if it contains spaces
+                    if($option =~ /\s/) {
```

Comment:
No, I do not need the indicies in the menu but maybe in a hash storing name and index so one can check and apply correctly. options are transmitted via /all endpoint or respective getters /getm odes etc.

I also do not need quoted white spaces (as this will not work) but changes to the names / entries by the use of make ReadingName($option).